Modules/System/Barcode/src/Barcode Provider 2D/Image/BarcodeImageProvider2D.enum.al

```diff
@@ -0,0 +1,22 @@
+// ------------------------------------------------------------------------------------------------
```

Comment:
I don't think we should add another set of symbologies for the image provider. Reuse the existing one. Your implementation only supports the QR symbology, but that is alright.

Reply:
I used the same symboligies. Or should I merge the providers?
enum 9206 "Barcode Font Provider 2D" and enum 9207 "Barcode Image Provider 2D" into one with two Implementations?
But this will be destructive if I rename the file to "Barcode Provider 2D".

Reply:
Sorry, that was a "brain fart". I'm still struggling a little to get the overview in this code review tool. My bad. All good here.